tensorflow/c/experimental/pluggable_device_utils/pluggable_device_utils.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Let's wrap functions here in `namespace pluggable_device` to avoid symbol clashes.

Reply:
done